app/views/layouts/application.html.erb

```diff
@@ -1,47 +1,42 @@
 <% content_for :body do %>
-  <% if use_govuk_account_layout? %>
-    <div id="wrapper">
-      <% content_for :before_content do %>
-        <%= yield :back_link %>
-      <% end %>
-      <%= yield :before_content %>
-      <main role="main" id="content">
-        <%= yield %>
-      </main>
-    </div>
-  <% else %>
-    <div class="govuk-width-container" id="wrapper">
-      <%= yield :back_link %>
-      <main class="govuk-main-wrapper<%= " govuk-main-wrapper--l" if yield(:back_link).blank?%>" role="main" id="content">
-        <div id="email-alert-frontend">
-          <div class="govuk-grid-row">
-            <div class="govuk-grid-column-two-thirds">
-              <%= yield %>
-            </div>
+  <div class="govuk-width-container" id="wrapper">
+    <%= yield :back_link %>
+    <main class="govuk-main-wrapper<%= " govuk-main-wrapper--l" if yield(:back_link).blank?%>" role="main" id="content">
+      <div id="email-alert-frontend">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            <%= yield %>
           </div>
         </div>
-      </main>
-    </div>
-  <% end %>
+      </div>
+    </main>
+  </div>
 <% end %>
 
-<!DOCTYPE html>
-<html>
-  <head>
-    <title><%= yield :title %> - GOV.UK</title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-    <% if content_for(:meta_description).present? %>
-      <meta name="description" content="<%= content_for(:meta_description) %>" />
-    <% end %>
-    <%= yield :head %>
-    <meta name="robots" content="noindex, nofollow">
-    <%=
-      render_component_stylesheets
-    %>
-  </head>
-  <body class="govuk-template__body">
-    <%= yield :body %>
-    <%= javascript_include_tag 'application', type: "module" %>
-  </body>
-</html>
+<%= content_for :head do %>
+  <%= csp_meta_tag %>
+  <meta name="robots" content="noindex, nofollow">
+
+  <% add_gem_component_stylesheet("cookie-banner") %>
+  <% add_gem_component_stylesheet("emergency-banner") %>
+  <% add_gem_component_stylesheet("feedback") %>
+  <% add_gem_component_stylesheet("global-banner") %>
+  <% add_gem_component_stylesheet("label") %>
+  <% add_gem_component_stylesheet("layout-header") %>
+  <% add_gem_component_stylesheet("layout-footer") %>
+  <% add_gem_component_stylesheet("layout-for-public") %>
+  <% add_gem_component_stylesheet("layout-super-navigation-header") %>
+  <% add_gem_component_stylesheet("phase-banner") %>
+  <% add_gem_component_stylesheet("search-with-autocomplete") %>
+  <% add_gem_component_stylesheet("skip-link") %>
+  <%= render_component_stylesheets %>
+<% end %>
+
+<%= render "govuk_publishing_components/components/layout_for_public", {
+      title: yield(:title),
+      show_explore_header: true,
```

Comment:
non-blocking: we should be able to delete this line now, the `show_explore_header` option was removed from layout_for_public in [v54.0.0](https://github.com/alphagov/govuk_publishing_components/blob/main/CHANGELOG.md#5400) of govuk_publishing_components